packages/webpack-cli/lib/groups/StatsGroup.js

```diff
@@ -13,20 +13,14 @@ class StatsGroup extends GroupHelper {
     }
 
     resolveOptions() {
-        if (this.args.verbose && this.args.stats) {
-            logger.warn('Conflict between "verbose" and "stats" options. Using verbose.');
-            this.opts.options.stats = 'verbose';
+        const { stats, json } = this.args;
+        if (stats && !StatsGroup.validOptions().includes(stats)) {
+            logger.warn(`'${stats}' is invalid value for stats. Using 'normal' option for stats`);
+            this.opts.options.stats = 'normal';
```

Comment:
We should not validate this, webpack do it for us :smile: Can we fix it here? And add tests with failing?

Reply:
yes need fixing, on it

Reply:
Done @evilebottnawi 👍